.github/workflows/clear-runs.yml

```diff
@@ -0,0 +1,88 @@
+name: Clear outdated runs
+
+on:
+  workflow_call:
+    secrets:
+      github-token:
+        description: The token needed for the reject outdated runs endpoint.
+        required: true
```

Comment:
Elsewhere we've used [assigning permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) for the default token.  Is this possible or less suitable here?

I believe the rationale for using permissions as per the link is that the permissions won't be accessible outside of this workflow (more secure) and it is 1 less secret to maintain.

Reply:
Unfortunately not, to approve/reject the workflow run the calling user has to be a required reviewer for that environment. I don't think we want to relax the current configurations of requiring TIS team members to approve, so we're stuck with using a PAT.